wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController+EmptyState.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireSyncEngine
+
+extension ConversationListViewController {
+
+    var isEmptyPlaceholderVisible: Bool {
+        let totalItems = listContentController.listViewModel.sections.map { $0.items.count }.reduce(0, +)
+        return totalItems == 0
+    }
+
+    var emptyPlaceholderForSelectedFilter: EmptyPlaceholder {
+        typealias Strings = L10n.Localizable.ConversationList.EmptyPlaceholder
+
+        guard let selectedFilter = listContentController.listViewModel.selectedFilter else {
+            return EmptyPlaceholder(
+                headline: Strings.All.headline + " 👋",
+                subheadline: Strings.All.subheadline.attributedString)
+        }
```

Comment:
should the emoji be localized?

Reply:
@netbe  Why? Do you mean send it to Crowdin?
This is per design:
```
Welcome 👋
Connect with others or create a new group to start collaborating!
```
I don't remember us having localized emojis.